playwright/test/view-items.test.ts

```diff
@@ -36,11 +42,64 @@ describe('Scenario 1: A user wants a large-scale view of an item', () => {
     await multiVolumeItem();
     await page.waitForSelector(fullscreenButton);
     await page.click(fullscreenButton);
-    // check full screen
+    // TODO: e.g. test for existence of document.fullscreenElement (can't figure out how to do this with Playwright)
     await page.waitForSelector(zoomInButton);
     await page.click(zoomInButton);
     await page.waitForSelector(openseadragonCanvas);
-    // expect(openseadragonCanvas).toBeTruthy();
+  });
+});
+
+describe('Scenario 2: A user wants to use the content offline', () => {
+  beforeEach(async () => {
+    await multiVolumeItem();
+    await page.waitForSelector(downloadsButton);
+    await page.click(downloadsButton);
+    await page.waitForSelector(itemDownloadsModal);
+  });
+
+  test('downloading an image of the current canvas', async () => {
+    const [newPage] = await Promise.all([
```

Comment:
This is nice, should it be a named function explaining what it does? Maybe it's not necessary yet, but feels like a pretty repeatable pattern.